### PR TITLE
[Notifications] - handle new type of notifications #1625

### PR DESCRIPTION
**Pull request by Illya97**
Follow the link: https://github.com/ita-social-projects/OoS-Frontend/issues/1625

### CODE REVIEW

File: src/app/shared/components/notifications/notifications-list/notifications-list.component.html [hunk 1]
```diff
@@ -22,7 +22,9 @@ <h3 class="title">СПОВІЩЕННЯ</h3>
   </ng-container>
   <ng-container *ngIf="notificationsData.notifications.length">
     <div *ngFor="let notification of notificationsData.notifications" matMenuItem disableRipple="true">
-      <span fxLayout="row wrap" fxLayoutAlign="space-between center">{{notification.text}}<mat-icon class="icon"
+      <span fxLayout="row wrap" fxLayoutAlign="space-between center">
+          Набір у гурток {{notification.data?.title}} {{(notification.data.Status=='Open')? 'відкрито': 'призупинено'}}
```
Comment (litvinets) [inline src/app/shared/components/notifications/notifications-list/notifications-list.component.html:26]: use enum WorkhopStatus

**Review by litvinets — CHANGES_REQUESTED** [review]

**Comment by Illya97** [comment]
image

File: src/app/shared/models/notifications.model.ts [hunk 2]
```diff
@@ -23,3 +23,8 @@ export interface Notifications {
   notificationsGrouped: NotificationGrouped[];
   notifications: Notification[];
 }
+
+export interface NotificationsDate {
+  Status: string;
+  Title: string;
```
Comment (litvinets) [inline src/app/shared/models/notifications.model.ts:29]: I believe status is not a required field

File: src/app/shared/enum/enumUA/workshop.ts [hunk 3]
```diff
@@ -13,4 +13,9 @@ export enum RecruitmentStatusUkr {
   RecruitmentStoped = 'Набір призупинено',
   Stop = 'Призупинити набір',
   Open = 'Відкрити набір'
+}
+
+export enum WorkshopStatusUkr {
```
Comment (litvinets) [inline src/app/shared/enum/enumUA/workshop.ts:18]: WorkshopNotificationStatusUkr

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**